packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -21,6 +22,32 @@ import {
 import S3 from "aws-sdk/clients/s3";
 import { performance } from "perf_hooks";
 import { ServerResponse } from "http";
+import jsonwebtoken from "jsonwebtoken";
+
+const NEXT_PREVIEW_DATA_COOKIE = "__next_preview_data";
+const NEXT_PRERENDER_BYPASS_COOKIE = "__prerender_bypass";
+
+const getPreviewCookies = (request: CloudFrontRequest) => {
+  const targetCookie = request.headers.cookie || [];
+  const previewCookie = targetCookie.find((cookieObj) => {
```

Comment:
I think using a `reduce` here instead of a map will save 2 the extra call to `cookie.parse` below.